### ISSUE TITLE
[feature] Multiple improvements related to options.html

### ISSUE THREAD

**Issue by Crystal-RainSlide** [BillDietrich/DecentraleyesSimpler#4]
- open `options.html` on clicking the menubar icon

- improve `options.html` itself, like add a `<title>`, replace `<font>`s with CSS?
  - I can help on the whole thing.

- Customize the indention level of the exported settings file

#### Use tab for indention:
``` JS
JSON.stringify(config, null, 1).replace(/^ +/mg, spaces => spaces.replace(/ /g, "\t"))
```
- Add a Setting Editor UI in the future?

**Comment by Crystal-RainSlide** [comment]
https://github.com/BillDietrich/DecentraleyesSimpler/blob/master/options.html

``` HTML
<!DOCTYPE html>

<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <title>Settings - Decentraleyes Simpler</title>
    <style>
:root {
  height: 100%;
}
body {
  margin: 0 auto;
  padding: 1rem;
  box-sizing: border-box;
}
@media (min-width: 80ch) {
  body {
    max-width: 80ch;
  }
}

.box {
  border: 2px solid black;
  padding: 1rem;
}
.box ~ .box {
  margin-block-start: 1rem;
}
.box:empty {
  display: none;
}

h1 {
  text-align: center;
}

.box > h3:first-child {
  margin-block-start: 0;
}

@media (prefers-color-scheme: dark) {
  :root {
    color: white;
    background-color: black;
  }
  .box {
    border-color: white;
  }
}
    </style>
  </head>

<body>

<h1>Settings - Decentraleyes Simpler</h1>

<div id="infodiv" class="box"></div>

<h2>Import and export</h2>

<div class="box">
  <h3>Export settings to file:</h3>
  <form id="exportform">
    <button type="submit" >Save As ...</button>
  </form>
</div>

<div class="box">
  <h3>Reset settings to default:</h3>
  <form id="resetform">
    <button type="submit" >Reset settings</button>
  </form>
</div>

<div class="box">
  <h3>Import settings from file:</h3>
  <p>WARNING: Importing settings deletes any existing settings and clears the cache.</p>
  <form id="importform">
    <input type="file" id="importfile" accept=".json" >
    <button type="submit" >Import settings</button>
  </form>
</div>

<script src="options.js"></script>

</body>

</html>
```

**Comment by BillDietrich** [comment]
Thanks for the ideas and code.  It's been a while since I worked on this extension, have to get my head back into it.  And now that I just updated node and web-ext, and Firefox has updated many times in last 8 months, stuff is broken in my debug environment.  Will try to straighten it out and then start making these changes.  Thanks.

**Comment by BillDietrich** [comment]
Addressed in release 2.0.1, please try it.

- Didn't change anything with the menu bar icon, I think it's good the way it is.

- Put in most of the CSS and HTML you gave, but I don't see much change in the UI.

- Made the JSON file have tabs instead of spaces.

**Comment by Crystal-RainSlide** [comment]
That's great. I will try to make a Setting Editor on it.

- the menu bar icon is not a big thing, I'm also OK with it.
- improvements in my code is just replace the `<br>`s with CSS and the [`prefers-color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) thing by now. Changes like more textual content, or redo the layout, will be put after the Setting Editor.
- I've got another solution with less regular expression, even less important than the menu bar icon, but if you would like to take it:

``` JavaScript
JSON.stringify(config, null, 1).replace(/^ +/mg, spaces => "\t".repeat(spaces.length))
```
